.github/workflows/prettier.yml

```
@@ -34,3 +34,25 @@ jobs:
+        with:
+          node-version: 18
+
+      - name: Install pnpm
+        uses: pnpm/action-setup@v2
+        with:
+          version: 8
+
++      - name: Install dependencies
++        run: pnpm install
+
+      - name: Lint
```

actionlint (1.7.9)

could not parse as YAML: mapping values are not allowed in this context

(syntax-check)